src/adapters/chineseloc/chinese-source/chinese-source.js

```diff
@@ -146,8 +155,10 @@ export default class ChineseSource {
 
           rs[count++] = finalResItem
         })
+        checkedWords = checkWords.length

```

Comment:
Actually this is wrong. We want to keep checking for additional words until the context forward is exhausted even if a result is found.

so, we should unconditionally set checkWords = checkedWords + 1 regardless of whether a result is found

Reply:
oh - I really didn't realize that we were taliking about several homonyms inside the request.
I was sure that it is about to find one homonym using context when the base targetWord is failed.

Reply:
Also all our Vuex attributes are created for one homonym at a time, not only vue-components.
I should think about - how it could be rearranged for several homonyms at a time.
Also it should be saved to Wordlist as several homonyms at a time.

@balmas it is really a huge rearrangement :)